src/lib/helpers.ts

```diff
@@ -1,5 +1,18 @@
+import {
+  PromiEvent,
+} from 'web3-core';
+import {
+  Contract,
+} from 'web3-eth-contract';
+
 import { USDC_ADDRESSES } from './Constants';
 
+export enum OUTCOMES {
+  INITIAL = 0,
+  RESOLVED = 1,
+  REJECTED = 2,
+}
```

Comment:
Is this only used for sending gasless transactions? Maybe note that in the enum name.

Also, can we use a better name than `INITIAL`? Maybe `PENDING`?

Reply:
That would change old logic which I don't want to do

Reply:
This is also used for normal sending which is why I didnt want to modify